<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="IN-PG0352AV" failures="0" tests="1" name="tests.ProductQuantityTest" time="0.000" errors="1" timestamp="07 May 2024 10:47:10 GMT" skipped="0">
  <testcase classname="tests.ProductQuantityTest" name="AddToCarttest" time="0.000">
    <error message="
Method public void tests.ProductQuantityTest.AddToCarttest(java.lang.String) throws java.io.FileNotFoundException,java.io.IOException,java.lang.InterruptedException requires a @DataProvider named : getProductQuantity(" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Method public void tests.ProductQuantityTest.AddToCarttest(java.lang.String) throws java.io.FileNotFoundException,java.io.IOException,java.lang.InterruptedException requires a @DataProvider named : getProductQuantity(
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:336)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:498)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- AddToCarttest -->
</testsuite> <!-- tests.ProductQuantityTest -->
